authorization-server/config/index.js

```diff
@@ -16,7 +16,7 @@
  */
 exports.token = {
   expiresIn               : 3600,
-  calculateExpirationDate : () => new Date(Date.now() + (this.expiresIn * 1000)),
+  calculateExpirationDate : () => new Date(Date.now() + (this.token.expiresIn * 1000)),
```

Comment:
I broke this in the 4.0.0 last commit.  Hopefully not many people got that version.